src/Maestro/maestro-angular/src/app/page/build/build.component.html

```diff
@@ -121,9 +125,33 @@ <h5 class="card-title float-left">
         </ng-template>
       </div>
 
+      <div class="container-fluid">
+        <div class="card mt-3">
+          <div class="card-body">
+            <h5 class="card-title float-left">
+              Subscriptions
+            </h5>
+            <!-- <span class="form-check float-right">
```

Comment:
Let's remove the commented out code if we aren't ready to enable this. I don't like the idea of checking in commented out code.

Reply:
Agreed - it'll either be in or out.